default.nix

```diff
@@ -61,7 +61,8 @@ let
         executableHaskellDepends = drv.executableHaskellDepends ++ [self.cabal-install];
       })));
       cardano-sl-node = addGitRev super.cardano-sl-node;
-      cardano-sl-wallet-new = addGitRev (justStaticExecutables super.cardano-sl-wallet-new);
+      cardano-sl-wallet-new = addGitRev super.cardano-sl-wallet-new;
+      cardano-sl-wallet-new-static = justStaticExecutables cardano-sl-wallet-new;
```

Comment:
`error: undefined variable 'cardano-sl-wallet-new' at /var/lib/buildkite-agent/builds/buildkite-agent-4/input-output-hk/cardano-sl/default.nix:65:60`

I believe you're just missing the `super.` here.

Reply:
yes, seems like extra `super.` here